device_samples/getting_started/src/send_messages/send_messages_to_central.js

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
nit: do you want to call this `send_message_to_iot_central.js`? also `send_mesage_to_iot_hub.js`